### ISSUE TITLE
Kernal driver prevents system to Hibernate

### ISSUE THREAD

**Issue by jnandani** [EaseFilter/FilterDriver#1]
Once installed EaseFlt.sys on Windows 10 / Laptop system, It won't allow to Hibernate the laptop.

**Comment by EaseFilter** [comment]
Did you block any IOs with the setting?

**Comment by jnandani** [comment]
No, just installed the driver, tested with demo application and closed the demo application.
